### ISSUE TITLE
should support literal search, duh

### ISSUE THREAD

**Issue by dthtvwls** [jessetane/unicodes#1]

**Comment by jessetane** [comment]
You're right, there definitely needs to be a way to do this! In my mind, search is for finding characters you can describe but don't have on hand, and urls are for literal lookups. That said, urls were previously hex strings which are next to useless for humans... so I've changed it - now copy and pasting emoji for example should work:

http://unicodes.jessetane.com/⛈

**Comment by jessetane** [comment]
Someone else asked me about this the other day. Maybe single character searches should be treated as lookups, but anything longer as an actual search?

**Comment by ppo** [comment]
It would be great to also work with character identifiers like…
- `U+203E` (or `203E`)
- UTF-* encoding: `0xE2 0x80 0xBE` (+on each part), `0x203E`, `0x0000203E`
- htmlentities: `&#8254`, `&#x203E`, `&oline`
- …

PS: And thank you for this great/useful tool 😀
